<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElGamalSerpentClass</class>
 <widget class="QMainWindow" name="ElGamalSerpentClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ElGamalSerpent</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="group">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century</family>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Выберите алгоритм&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="keyGen">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>146</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сгенерировать ключи</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>92</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="ElGamal">
        <property name="font">
         <font>
          <family>Century</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ElGamal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="Serpent">
        <property name="font">
         <font>
          <family>Century</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Serpent</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>431</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSplitter" name="splitter_5">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>100</y>
       <width>419</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Century</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберите путь для сохранения</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOutputFile">
      <property name="font">
       <font>
        <family>Century</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openOutputFile">
      <property name="font">
       <font>
        <family>Century</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
    </widget>
    <widget class="QSplitter" name="splitter_4">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>70</y>
       <width>313</width>
       <height>23</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Century</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выберите файл</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineInputFile">
      <property name="font">
       <font>
        <family>Century</family>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openInputFile">
      <property name="font">
       <font>
        <family>Century</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Обзор</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="deleteFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>178</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить исходный файл</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>145</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="encryption">
        <property name="font">
         <font>
          <family>Century</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Шифрование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="decryption">
        <property name="font">
         <font>
          <family>Century</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дешифрование</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>133</width>
       <height>49</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="selectModeLabel">
        <property name="font">
         <font>
          <family>Century</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Выберите режим</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="modes">
        <property name="font">
         <font>
          <family>Century</family>
         </font>
        </property>
        <item>
         <property name="text">
          <string>ECB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CTR</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CBC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CFB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OFB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>RandomDelta</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>118</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Введите ключи</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushStart">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Запуск</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="elgamalPrimeNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>prime number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="elgamalRoot">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>root</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="elgamalSecretKey">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>secret key</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="elgamalY">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serpentKey">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>key</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serpentIV">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>132</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>IV</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ElGamalSerpent.qrc"/>
 </resources>
 <connections/>
</ui>
